### ISSUE TITLE
AboutYou.html

### ISSUE THREAD

**Issue by pallavikotturu** [pallavikotturu/HW8#3]
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8" />
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>About You </title>
</head>
<body>
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="container">
            <div class="navbar-header">
                <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>

                <img src="Images/Logo.PNG" class="ui-icon-image" height="50" width="50" />
            </div>

            <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
                <ul class="nav navbar-nav">
                    <li><a href="Home.html">Home</a></li>
                    <li><a href="Login.html">Login</a></li>
                    <li class="active"><a href="AboutYou.html">About You</a></li>
                </ul>
            </div>
        </div>
    </div>

    <div class="container" style="margin-top: 3%">
        <form class="form-horizontal">
            <fieldset>
                
                <div style="margin-left: 25%"><h2>Please tell us a little bit about you to get started</h2></div>
                <hr />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="f_name">First Name</label>
                    <div class="col-md-6">
                        <input id="f_name" name="f_name" type="text" placeholder="Enter Your First Name " class="form-control input-md" required="">
                    </div>
                </div>
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="l_name">Last Name</label>
                    <div class="col-md-6">
                            <input id="l_income" name="l_income" class="form-control" placeholder="Enter Your Last Name" type="text" required="">
                    </div>
                </div>
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="address">Address</label>
                    <div class="col-md-6">
                        <div class="input-group">
                            <span class="input-group-addon">Address</span>
                            <input id="address" name="address" type="text" placeholder="Enter Address" class="form-control input-md" required="">
                        </div>
                        </div>
                </div>
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="city">City</label>
                    <div class="col-md-6">
                        <div class="input-group">
                            <span class="input-group-addon">City</span>
                            <input id="city" name="city" class="form-control" placeholder="Enter City Name" type="text" required="">
                        </div>
                    </div>
                </div>
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="state">State</label>
                    <div class="col-md-6">
                        <div class="input-group">
                            <span class="input-group-addon">State</span>
                            <input id="state" name="state" class="form-control" placeholder="Enter State Name" type="text" required="">
                        </div>
                    </div>
                </div>

                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="zip">Enter Zipcode</label>
                    <div class="col-md-6">
                        <input id="zip" name="zip" type="text" placeholder="#####" type="number" required="">
                    </div>
                </div>

                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="email">Enter Email</label>
                    <div class="col-md-6">
                        <input id="email" name="email" type="text" placeholder="username@me.com" type="email" required="">
                    </div>
                </div>

                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="bday">Enter Birthday</label>
                    <div class="col-md-6">
                        <input id="bday" name="bday" type="text" placeholder="MM-DD-YYYY" type="date" required="">
                    </div>
                </div>
               
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="phoneNumber">Enter Phone Number</label>
                    <div class="col-md-6">
                        <input id="phoneNumber" name="phoneNumber" type="text" placeholder="###-###-####" type="tel" required="">
                    </div>
                </div>

                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="zip">Enter Questions or Comments (Optional): </label>
                    <div class="col-md-6">
                        <input id="comments" name="comments" type="text" placeholder="text" type="text">
                    </div>
                    <br/>
                    <br/>
                </div>

                <button style="margin-left: 34%" type="submit" class="btn btn-sm btn-success">Create My Account</button>

            </fieldset>
        </form>
    </div>

</body>
</html>
